src/common/components/NewsEntriesList/NewsEntry.tsx

```diff
@@ -0,0 +1,98 @@
+import { Box, SxProps, Typography } from '@mui/material';
+import ErrorRounded from '@mui/icons-material/ErrorRounded';
+
+const ENTRY_DESCRIPTION_LENGTH = 75;
+
+interface NewsEntryProps {
+  title: string;
+  categoryColor: string;
+  isAcceptanceMandatory: boolean;
+  content: string;
+  createdAt: string;
+  isSeen: boolean;
+  isConfirmed: boolean;
+  isActive: boolean;
+  handleSelection: () => void;
+}
+
+export const NewsEntry = ({
+  title,
+  categoryColor,
+  isAcceptanceMandatory,
+  content,
+  createdAt,
+  isSeen,
+  isConfirmed,
+  isActive,
+  handleSelection
+}: NewsEntryProps) => {
+  const shortenedContent = content.substring(0, ENTRY_DESCRIPTION_LENGTH) + '...';
+
+  return (
+    <Box sx={entryContainerStyle(isSeen, isActive, isAcceptanceMandatory, isConfirmed)} onClick={handleSelection}>
+      <Box sx={{ width: '100%' }}>
+        <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
+          <Box sx={{ display: 'flex', alignItems: 'center', width: '275px' }}>
+            {isAcceptanceMandatory && !isConfirmed && (
+              <ErrorRounded sx={{ color: '#454749', pr: (theme) => theme.spacing(2) }} />
+            )}
+            <Typography component="h3" sx={entryTitleStyle(isSeen)}>
+              {title}
+            </Typography>
+          </Box>
+          <Typography component="h3" sx={entryTimestampStyle(isSeen)}>
+            {createdAt}
+          </Typography>
+        </Box>
+        <Box sx={entrySeparator(categoryColor)} />
+        <Typography component="p" sx={{ fontSize: '14px', width: '290px' }}>
```

Comment:
```suggestion
        <Typography variant="body1">
```
or body2 it depends which one matches requirements